ollama-model-service/docs/models.md

```diff
@@ -0,0 +1,31 @@
+# Supported Models
+
+The following models are accessible via this model server API.
+
+## IBM Granite 20B Code Instruct
+
+[Hugging Face Repository](https://huggingface.co/ibm-granite/granite-20b-code-instruct)
+
+<!--
+TechDocs read this as a URL starting from where the docs are located, so we can use ../ behaviour to navigate the TechDocs to reference other resources/components/apis
+-->
+
+[Model Documentation](../../../../default/resource/granite-20b-code-instruct)
```

Comment:
All of the `../` techdoc references in this file double up on `default` and don't send you to the right page

Reply:
Switching to absolute paths seemed to fix this